kong/plugins/oauth2/access.lua

```diff
@@ -88,6 +88,10 @@ do
   end
 end
 
+local function get_plugin_identifer(conf)
```

Comment:
this whole has a repeated typo everywhere ... it's identif_i_er, not "identifer" ...

I would normally not make a huge deal about this for private ids, but this name is public.